devicetypes/smartthings/zigbee-stateless-curtain-window-shade.src/zigbee-stateless-curtain-window-shade.groovy

```diff
@@ -0,0 +1,112 @@
+/**
+ *
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+import physicalgraph.zigbee.zcl.DataType
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+metadata {
+    definition(name: "Zigbee Stateless Curtain Window Shade", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-curtaion") {
```

Comment:
If the device sends level reports back to us, is it really stateless?

Reply:
I would suggest that we call this DTH `ZigBee Curtain` or `ZigBee Window Shade`. This will keep consistent with our naming convention.

Also, the vid `generic-curtaion` has a typo. I don't think that `generic-curtain` exists right now.